src/fstests/runner.py

```diff
@@ -54,21 +85,17 @@ def run(self):
             self._db.unsubscribe(sub_id)
             return True
 
-class cmd_check(Command):
-    help = 'Check KVM-xfstests environment'
-    args = [Args.db_config]
-    opt_args = [Args.verbose]
-
-    def __call__(self, configs, args):
-        return FstestsRunner(configs, args).check_kvm_xfstest_env()
 
 class cmd_run(Command):
     help = 'KVM fstests runner'
-    args = [Args.db_config]
+    args = [
+        Args.db_config, Args.lab_config, Args.output,
+        Args.plan, Args.target,
```

Comment:
We don't need `lab_config`, `plan` and `target`.  This service is specific to `fstests`.

Reply:
ok, all set with constant/hardcoded..